server/package.json

```diff
@@ -33,6 +34,7 @@
     "dotenv": "^16.4.5",
     "dotenv-cli": "^7.4.2",
     "jose": "^5.8.0",
+    "lib_test": "file:",
```

Comment:
What's this? Remove it, if it's unnecessary. Explain here if it's necessary.

Reply:
It's not necessary, I will remove that.